content/en/references/configuration.md

```diff
@@ -78,7 +78,7 @@ This section covers configuration values that are specific to certain AWS servic
 
 | Variable | Example Values | Description |
 | - | - | - |
-| `GRAPHQL_ENDPOINT_STRATEGY` | `legacy`\|`domain`\|`path` |  Governs how AppSync endpoints are created to access a GraphQL API (see [AppSync Endpoints]({{< ref "appsync#endpoints" >}})) |
```

Comment:
This is not related to the current PR. It is a fix for a build error which I was getting at this point saying "unambiguous reference" and I fixed it by giving the full path.

In developer hub, when I created an application with appsync tag, build started to fail and this error popped up

@lukqw